interface/wx/ribbon/bar.h

```diff
@@ -462,6 +462,57 @@ class wxRibbonBar : public wxRibbonControl
     */
     wxRibbonDisplayMode GetDisplayMode() const;
 
+    /**
+        Returns whether the bar contains multiple pages.
+
+        The default implementation always returns @c true.
+
+        @since 3.1.0
+    */
+    virtual bool HasMultiplePages() const;
+
+    /**
+        Returns @c true if the toggle button is currently moused over.
+
+        @since 3.1.0
+    */
+    bool IsToggleButtonHovered() const;
+
+    /**
+        Returns @c true if the help button is currently moused over.
+
+        @since 3.1.0
+    */
+    bool IsHelpButtonHovered() const;
+
+    /**
+        Hide the panels if its display mode is @c wxRIBBON_BAR_EXPANDED.
+
+        @since 3.1.0
+    */
+    void HideIfExpanded();
+
+    /**
+        Sets the wxRibbonBarOption flags for the ribbon bar.
+
+        @since 3.1.0
+    */
+    void SetWindowStyleFlag(long style);
+
+    /**
+        Returns the ribbon bar's wxRibbonBarOption flags.
+
+        @since 3.1.0
+    */
+    long GetWindowStyleFlag() const;
```

Comment:
Those are inherited from wxWindow, is there a good reason to redocument them here?